.github/workflows/pr.yml

```diff
@@ -0,0 +1,27 @@
+name: PR Workflow
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v2
+      with:
+        node-version: '15.x'
+    
+    - name: Install dependencies
+      run: npm ci
+    
+    - name: Bootstrap
+      run: npm run bootstrap  
+
+    - name: Lint types and code
```

Comment:
"Lint types" sounds odd. Can you rephrase a little bit please?

Reply:
Is `Lints and formats code` better?

I copied the `Lint types and code` from Chakra, I thought it was a community definiton.

Reply:
Or maybe just `Lint`

Reply:
> Is Lints and formats code better? 

We arent doing _any_ formating here. We only checking if a developer is pushing code that it isnt following our eslint rules

Reply:
Just Lint sounds good to me